src/api/types.ts

```diff
@@ -0,0 +1,85 @@
+export interface Exercise {
+    [key: string]: string | string[] | undefined | '' | 'kg' | 'lb',
+    name: string,
+    img: string,
+    muscles?: string[],
+    sets?: string,
+    reps?: string,
+    weight?: string,
+    unit?: "" | "kg" | "lb",
```

Comment:
Can unit be an empty string?

Reply:
Yeah, it is at the beginning until the user selects the unit/creates exercise, like so:
```js
const exerciseData = ref<Exercise>({
  name: '',
  img: '',
  muscles: [],
  sets: '',
  reps: '',
  weight: '',
  unit: '',
  rest: ''
})
```